x/interchainstaking/types/codec.go

```diff
@@ -39,8 +39,8 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 func init() {
 	cryptocodec.RegisterCrypto(amino)
 	govtypes.RegisterProposalType(ProposalTypeRegisterZone)
-	govtypes.RegisterProposalTypeCodec(&RegisterZoneProposal{}, "cosmos-sdk/RegisterZoneProposal")
+	amino.RegisterConcrete(&RegisterZoneProposal{}, "cosmos-sdk/RegisterZoneProposal", nil)
```

Comment:
https://github.com/notional-labs/evmos/blob/dang/fix-sdk46-test/x/erc20/types/proposal.go#L33

Reply:
try to run and will face amino is sealed

Reply:
Fix in https://github.com/notional-labs/quicksilver/commit/aab37f851670fde663ed1b04b299e6be033aed8d